javascript/js-modern-syntax/js-modern-syntax_destructuring-objects/index.js

```diff
@@ -41,6 +47,8 @@ const person = {
   firstName: "Alex",
 };
 
+const { lastName: personLastName, ...allOtherInfo } = person;
```

Comment:
You did fine here. The only issue here is that the task is to *'Store all other properties in an object called "moreInformation".'* So the test is failing because of the incorrect variable name. But the output is correct..no matter what variable name is used here. So it's a pedantic change request here :)

~~`{ lastName: personLastName, ...allOtherInfo }`~~ ➔ `{ lastName: personLastName, ...moreInformation }`